src/action.js

```diff
@@ -5,18 +5,20 @@ import { ACTION_STATES } from './constants'
 import { inherit } from './utils'
 
 /**
- * Actions encapsulate the process of resolving an action creator. Create an
- * action using `Microcosm::push`:
+ * Actions encapsulate the process of resolving an action
+ * creator. Create an action using `Microcosm::push`:
  */
-export default function Action (command, history) {
+export default function Action (command, history, status) {
   Emitter.call(this)
 
-  this.command = tag(command)
   this.history = history || new History()
+
+  this.id = this.history.getId()
+  this.command = tag(command)
+  this.status = status || 'inactive'
```

Comment:
Actually, this is already set as the default. I should just do `if(status) this.status = status`